.buildkite/scripts/skip_if_only_docs_changes.sh

```diff
@@ -0,0 +1,17 @@
+# Source this file rather than fork-exec-ing it.
+
+only_docs_changes() {
+  test -n "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" &&
+    git diff --quiet "refs/remotes/origin/$BUILDKITE_PULL_REQUEST_BASE_BRANCH.." -- \
+      ':(exclude).changelog' \
+      ':(exclude)CHANGELOG.md' \
```

Comment:
I would replace all these with `:(exclude)*.md` to cover existing and new Markdown files anywhere (we have many README.md files in various subtrees).

Reply:
ok